### PR TITLE
SCMOD-3144: Updated to base off opensuse-base-image

### PR DESCRIPTION
**Pull request by michael-bryson**
https://jira.autonomy.com/browse/SCMOD-3144

### CODE REVIEW

File: release-notes-1.3.0.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Version Number
+- This image is now based off the [OpenSUSE Base Image](https://github.com/CAFapi/opensuse-base-image) which has the postgreSQL client pre-installed. This provides the ability to create databases during startup of any consuming containers.
```
Comment (dermot-hardy) [inline release-notes-1.3.0.md]: Version number token shouldn't be being removed.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

File: release-notes-1.3.0.md (hunk 1 above)
Comment (michael-bryson) [reply release-notes-1.3.0.md]: Sorry my mistake I put the text in the wrong place.

**Comment by michael-bryson** [comment]
Fixed release notes

**Review by dermot-hardy — CHANGES_REQUESTED** [review]
Can we do something like this:
https://github.com/CAFapi/opensuse-tomcat-image/blob/v2.0.0/release-notes-2.0.0.md

Rather than just saying we are using a new base image list the new features from the base image in a sub-section.  After all the reader doesn't overly care which level the features come in at.

**Comment by michael-bryson** [comment]
Updated release notes as requested

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**